andela/ah-aquaman-frontend
# #164069195 bookmark articles

## What does this PR do?

This PR implements the feature of a user being able to bookmark or unbookmark an article.

## Description of the task to be completed

- Enable users to bookmark an article
- Enable users to unbookmark an article

## How should this be manually tested?

- Clone this repository
- cd into the directory Ah-aquaman-frontend
- Check out the branch `ft-bookmark-articles-164069195`
- Run `npm install` to install all dependencies
- Run`npm start` to get the server running
- Once you are logged in, you can view an article and bookmark it but you can not bookmark your own article
- You can also remove a bookmark from an article you had previously bookmarked

## What are the relevant pivotal tracker stories?

[#164069195](https://www.pivotaltracker.com/story/show/164069195)

## Screenshots

An article that has been bookmarked

![image](https://user-images.githubusercontent.com/42435346/56674421-b18ce400-66c2-11e9-9c8c-249bb091b561.png)

An article that has been unbookmarked

![image](https://user-images.githubusercontent.com/42435346/56674464-c8333b00-66c2-11e9-83f3-2b2e548f6946.png)
